libs/template/package.json

```
@@ -49,9 +49,9 @@
     "url": "https://github.com/rx-angular/rx-angular.git"
   },
   "peerDependencies": {
-    "@angular/core": ">=12.0.0",
-    "@angular/platform-browser": ">=12.0.0",
-    "@rx-angular/cdk": ">=1.0.0",
+    "@angular/core": ">=13.0.0",
+    "@angular/platform-browser": ">=13.0.0",
```

issue: Only mark Angular 13.x as officially supported
```suggestion
    "@angular/core": "^13.0.0",
    "@angular/platform-browser": "^13.0.0",
```

---

IMO we should manage compat range this way: 

RxAngular 2.0.0 :arrow_right: @angular/core >=13.0.0
RxAngular 14.0.0 :arrow_right: @angular/core ^14.0.0